### PR TITLE
Connect to VPN: Add a menu bar command

### PR DESCRIPTION
**Pull request by vpukhanov**
## Description

Hey folks!

I've been using this extension frequently and found myself toggling the VPN on and off multiple times a day. Navigating through the keyboard commands became quite tedious, so I decided to add a menu bar option for easier control. This pull request introduces a menu bar extra, allowing VPN connections to be managed directly from the menu bar.

Here's what I changed in this Pull Request:

- Ensured all extension dependencies are up-to-date
- Renamed `src/index.tsx` to `src/view.tsx` for the 'view' command
- Added a new menu bar command implemented in `src/menu-bar.tsx`
- Extracted common logic from the commands into a shared module `src/network-services.ts`
- Moved status-to-action mapping and hardware port normalization for subtitles into the shared module

I’ve organized my changes into separate commits, but it seems `@raycast/api publish` squashed them into a single commit.

Please let me know if you need any adjustments or if there's anything else I can do!

Thanks!

## Screencast

https://github.com/raycast/extensions/assets/1694616/6003ec44-54dd-4610-938c-513042aa319a

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by rasmusbe — APPROVED** [review]
Tried it and works well, thanks for your contribution!

File: extensions/connect-to-vpn/package.json [hunk 1]
```diff
@@ -7,16 +7,24 @@
...
     "sato11",
-    "stonerl"
+    "stonerl",
+    "vyacheslav"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "view",
```
Comment (pernielsentikaer) [inline extensions/connect-to-vpn/package.json:16]: Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

Comment (vpukhanov) [reply extensions/connect-to-vpn/package.json:16]: Definitely! Wasn't aware it works like that. I'll make the change this evening 👍🏻

Comment (vpukhanov) [reply extensions/connect-to-vpn/package.json]: Done ✅

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**